keyboards/scottokeebs/scotto34/keymaps/scotto/config.h

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Would you be able to:
1. remove the `WS2812_DI_PIN` and `RGBLED_NUM` definitions, then
2. move this file into `default` keymap directory

Reply:
Done in a662ed2.